CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+<!-- SPDX-License-Identifier: CC0-1.0 -->
+# Contributing to Govdirectory
+
+🙇‍♀️ Thank you for contributing!
+
+We love issues and pull requests from everyone.
+If you're not comfortable with those features of GitHub, you can also just [start a discussion](https://github.com/govdirectory/website/discussions).
+
+## Problems, suggestions and questions in issues
+
+Please help development by reporting problems, suggesting changes and asking questions.
+To do this, you can [create a GitHub issue](https://help.github.com/articles/creating-an-issue/) for this project in the [GitHub Issues for the website](https://github.com/govdirectory/website/issues).
+
+You don't need to change any of our code or documentation to be a contributor!
+
+## Documentation and code in pull requests
+
+If you want to add to the documentation or code of one of our projects you should make a pull request.
+
+If you never used GitHub, get up to speed with [Understanding the GitHub flow](https://guides.github.com/introduction/flow/) or follow one of the great free interactive courses in the [GitHub learning lab](https://lab.github.com/) on working with GitHub and working with MarkDown, the syntax this project's documentation is in.
```

Comment:
None of these links actually explains how to create a pull request nor the common pitfalls. We should instead link to the ["Github Guides"](https://guides.github.com/) and in particular the ["Forking" guide](https://guides.github.com/activities/forking/) while mentioning that one should always create a new branch within ones fork.

Reply:
That's a good suggestion! Can you phrase it nicely?